week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -9,10 +8,21 @@
 
   Use the tests to confirm which message to return
 */
-
-function bushChecker() {
-
+function checkColor(colour){
+    if(colour === "pink"){
+      return colour;
```

Comment:
This is really close!

You should actually return true or false from this function so that it the function below works in all cases.

Technically this is working because you compare `result` to what is returned below and a none empty string will return null.